api/controllers/subscriber.js

```diff
@@ -69,23 +70,34 @@ async function getSubscriber(req, res) {
   Subscriber.findOne({ userId: userId }, async function (err, subscriber) {
     if (err) {
       return res.status(409).json({
+        success: false,
         message: 'Error getting subscriber',
         error: err.message,
       });
     }
     if (!subscriber) {
-      return res.status(404).json({
-        message: 'Error getting subscriber',
-        error: 'subscriber not found',
-      });
+      if (req.headers.newversion) {
```

Comment:
I think we could use numbers to set a value for the version. newversion only gives us 1 chance to versioning.